### PR TITLE
Update document

### PR DESCRIPTION
**Pull request by LiliDeng**
Update document for configuration file format and add one sample for how to run test against HYPERV

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,26 +93,33 @@ Please follow the steps mentioned at [here](https://docs.microsoft.com/en-us/azu
...
-            <DestinationOsVHDPath>VHDs_Destination_Path</DestinationOsVHDPath>
-        </Host>
+        <Hosts>
+            <Host>
+                <ServerName>localhost</ServerName>
+                <SourceOsVHDPath></SourceOsVHDPath>
+                <DestinationOsVHDPath>VHDs_Destination_Path</DestinationOsVHDPath>
+            </Host>
+            <Host>
+                <ServerName>localhost</ServerName>
```
Comment (juhlee-microsoft) [inline README.md]: Let's put another host name like lis-3345.

File: README.md [hunk 2]
```diff
@@ -93,26 +93,33 @@ Please follow the steps mentioned at [here](https://docs.microsoft.com/en-us/azu
...
+                <SourceOsVHDPath></SourceOsVHDPath>
+                <DestinationOsVHDPath>VHDs_Destination_Path</DestinationOsVHDPath>
```
Comment (juhlee-microsoft) [inline README.md]: Can we add an example path in comment?

File: README.md [hunk 3]
```diff
@@ -93,26 +93,33 @@ Please follow the steps mentioned at [here](https://docs.microsoft.com/en-us/azu
...
 
 3. There are two ways to run LISAv2 tests:
 
    a. Provide all parameters to Run-LisaV2.ps1
-   
+
         .\Run-LisaV2.ps1 -TestPlatform "Azure" -TestLocation "<Region location>" -RGIdentifier "<Identifier of the resource group>" [-ARMImageName "<publisher offer SKU version>" | -OsVHD "<VHD from storage account>" ] [[-TestCategory "<Test Catogry from Jenkins pipeline>" | -TestArea "<Test Area from Jenkins pipeline>"]* | -TestTag "<A Tag from Jenkins pipeline>" | -TestNames "<Test cases separated by comma>"]
-        
+        .\Run-LisaV2.ps1 -TestPlatform "HyperV " [-TestLocation "ServerName"] -RGIdentifier "<Identifier of the vm group>" -OsVHD "<local or UNC path>" [[-TestCategory "<Test Catogry from Jenkins pipeline>" | -TestArea "<Test Area from Jenkins pipeline>"]* | -TestTag "<A Tag from Jenkins pipeline>" | -TestNames "<Test cases separated by comma>"]
+
```
Comment (juhlee-microsoft) [inline README.md]: Add examples

**Comment by juhlee-microsoft** [comment]
@chvalean @adriansuhov  From another email thread, we find new document update for missing steps in Hyper-V setup. Adrian, can you add those missing steps here like External vSwitch, memory size, etc?

File: README.md (hunk 3 above)
Comment (LiliDeng) [reply README.md]: @juhlee-microsoft Thanks for review, cloud you please let know I need add examples for both azure and hyperv or just hyperv is enough?

**Comment by juhlee-microsoft** [comment]
Both please.

________________________________

**Comment by juhlee-microsoft** [comment]
@LiliDeng I signed off. Let's wait for other's comment.

**Comment by LiliDeng** [comment]
@juhlee-microsoft Thanks Ju.

**Comment by chvalean** [comment]
> @chvalean @adriansuhov From another email thread, we find new document update for missing steps in Hyper-V setup. Adrian, can you add those missing steps here like External vSwitch, memory size, etc?

Those relate to the environment one-time setup that might be require. But those don't make subject to how lisav2 is run.
Probably a new document would be required to detail how to setup a new Hyper-V server to be ready to be used in testing.

**Comment by juhlee-microsoft** [comment]
Create new issue #399. Let's review if it needs new document or a new chapter only.

**Pull request merged**